packages/legend-application-studio/src/components/editor/edit-panel/mapping-editor/PropertyMappingsEditor.tsx

```diff
@@ -188,7 +235,27 @@ export const PropertyMappingsEditor = observer(
         })}
       >
         <div className="property-mapping-editor__metadata">
-          <div className="property-mapping-editor__name">{property.name}</div>
+          <div
+            className={clsx('property-mapping-editor__name--with-validation', {
+              'property-mapping-editor__name--with-validation-error': Boolean(
+                validationErrorMessage,
+              ),
+            })}
+          >
+            <div
+              className={'property-mapping-editor__name--with-validation__name'}
+            >
+              {property.name}
+            </div>
+            {validationErrorMessage && (
+              <div
+                className="property-mapping-editor__name--with-validation__error"
+                title={validationErrorMessage}
+              >
+                <TimesCircleIcon />
```

Comment:
The condition for showing this remove button is also something to consider about:
1. Only shows it for non-class properties
2. Only shows this when there's duplicates

Reply:
not consistent though, delete means removing an entry in some places and clearing the property mapping in some other places would be confusing imo